system/lib/standalone/standalone.c

```diff
@@ -32,6 +38,13 @@
 
 // libc
 
+#ifdef NDEBUG
+#define REPORT_UNSUPPORTED(action)
+#else
+#define REPORT_UNSUPPORTED(action) \
+  emscripten_console_error("the program tried to " #action ", this is not supported in standalone mode, the program will exit now...\n");
```

Comment:
You don't need the tailing newline here.

Also, maybe just drop the `, the program will exit now...` part.

Reply:
Updated